### ISSUE TITLE
twitter Link | Title

### ISSUE THREAD

**Issue by elham-rababah** [yungzhu/flutter_link_preview#12]
The package determine this as title

JavaScript is not available. We’ve detected that JavaScript is disabled in this browser. Please enable JavaScript or switch to a supported browser to continue using twitter.com. You can see a list of supported browsers in our Help Center. Help Center Terms of Service Privacy Policy Cookie Policy Imp

**Comment by KennyChan94P** [comment]
yes, im having the same issue, is there any workarounds or other plugin recommendations?

**Comment by elham-rababah** [comment]
@KennyChan94P  as workarounds you can use flutterwebView to show the post from twitter
https://pub.dev/packages/webview_flutter
